hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/acl/OMVolumeSetAclRequest.java

```diff
@@ -88,6 +92,12 @@ public String getVolumeName() {
     return volumeName;
   }
 
+  @Override
+  OzoneObjInfo getObjectInfo() {
+    return OzoneObjInfo.fromProtobuf(
+        getOmRequest().getSetAclRequest().getObj());
```

Comment:
NIT: this can be put into the constructor with a member variable so that the protobuf conversion is done only once and the getter() simply return the member variable. Same apply to other places.

Reply:
fixed.